src/core/public/doc_links/doc_links_service.ts

```diff
@@ -36,7 +35,8 @@ export class DocLinksService {
         settings: `${ELASTIC_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/settings.html`,
         elasticStackGetStarted: `${STACK_GETTING_STARTED}get-started-elastic-stack.html`,
         upgrade: {
-          upgradingElasticStack: `${STACK_DOCS}upgrading-elastic-stack.html`,
+          upgradingStackOnPrem: `${ELASTIC_WEBSITE_URL}guide/en/elastic-stack/8.0/upgrading-elastic-stack-on-prem.html`,
```

Comment:
We should actually be pointing to 8.0, not the current version. There is also a doc page now dedicated to on-prem.